### DISCUSSION TITLE
M:N Ids to Url mappings

### DISCUSSION THREAD

**Discussion by bestia-dev** [crev-dev/cargo-crev#439] [Q&A]
I am wondering why is it possible to have more "crev ids" for one repository url?
Is there a need for this? What use case am I not aware of?

**Comment by dpc** [comment]
An id can change their url and start advertizing new one in new proofs, to allow smooth transition of proof hosting, etc.

**Comment by dpc** [comment]
Eg. if I moved my proof repo gitlab, I could change my url, publish to both old and new one, and eventually retired the old repo and delete it.

Or at least that's the theory.

**Comment by dpc** [comment]
Oh, sorry. I've misunderstood. The other way around. 

One repository can host multiple ids - people can share a repository (eg. in organization) or have different ids for professional and personal use etc.
